providers/os/config/config.go

```diff
@@ -251,6 +292,12 @@ var Config = plugin.Provider{
 			Aliases: []string{"fs"},
 			Use:     "filesystem PATH [flags]",
 			Short:   "a mounted file system target",
+			Long: `Use the filesystem provider to query mounted file systems. 
+
+Examples:
+  cnspec scan filesystem --path <MOUNT-PATH-TO-FILE-SYSTEM>
+	cnquery shell fs --path <MOUNT-PATH-TO-FILE-SYSTEM>
```

Comment:
Should these be 
        cnspec scan filesystem <MOUNT-PATH-TO-FILE-SYSTEM>
	cnquery shell fs <MOUNT-PATH-TO-FILE-SYSTEM>
??
I'm not sure if --path is for something different than the mount path.